lib/hooks/useNavItems.tsx

```diff
@@ -59,7 +67,7 @@ export default function useNavItems(): ReturnType {
       text: 'Blocks',
       nextRoute: { pathname: '/blocks' as const },
       icon: blocksIcon,
-      isActive: pathname === '/blocks' || pathname === '/block/[height_or_hash]',
+      isActive: (pathname === '/blocks' || pathname === '/block/[height_or_hash]') && !query.tab,
```

Comment:
This was when I added the Reorgs nav link, because we wanted to highlight Reorgs link instead of Blocks link, but we ended up using the tab system for that, so removing this extra condition I added